docs/core/preview3/tools/dotnet-new.md

```diff
@@ -4,56 +4,103 @@ description: The dotnet-new command creates new .NET Core projects in the curren
 keywords: dotnet-new, CLI, CLI command, .NET Core
 author: blackdwarf
 ms.author: mairaw
-ms.date: 10/12/2016
+ms.date: 02/15/2017
 ms.topic: article
 ms.prod: .net-core
 ms.technology: dotnet-cli
 ms.devlang: dotnet
 ms.assetid: fcc3ed2e-9265-4d50-b59e-dc2e5c190b34
 ---
-
 #dotnet-new (.NET Core Tools RC4)
 
-> [!WARNING]
-> This topic applies to .NET Core Tools RC4. For the .NET Core Tools Preview 2 version,
-> see the [dotnet-new](../../tools/dotnet-new.md) topic.
+[!INCLUDE[preview-warning](../../../includes/warning.md)]
 
 ## Name
-dotnet-new -- Creates a new .NET Core project in the current directory.
+dotnet-new - Creates a new .NET Core project in the current directory
 
 ## Synopsis
-`dotnet new [--help] [--type] [--lang]`
+```
```

Comment:
the options are specific to the templates and not to `dotnet new`. There may be additional templates installed using `dotnet new --install`.

It is mentioned in the doc here
>## Template options
>Each project template may have additional options available. The core templates, for >example, have the following.